pootle/apps/pootle_translationproject/models.py

```diff
@@ -191,11 +191,15 @@ def fullname(self):
 
     @property
     def abs_real_path(self):
-        return absolute_real_path(self.real_path)
+        if self.real_path is not None:
+            return absolute_real_path(self.real_path)
```

Comment:
@phlax No, I guess I only want that for the ones you want to get landed. Now seriously, lets try to get code landed in the better shape possible.